requirements.txt

```diff
@@ -2,6 +2,7 @@ appdirs~=1.4
 coala_utils~=0.6.2
 colorlog~=2.7
 dependency_management~=0.3.1
+# ipdb
```

Comment:
@userzimmermann maybe add some text like `# ipdb optional for debugging` so it gets clear